eclair-core/src/test/scala/fr/acinq/eclair/payment/PaymentRequestSpec.scala

```diff
@@ -291,6 +297,38 @@ class PaymentRequestSpec extends FunSuite {
     assert(pr.features.bitmask === BitVector.empty)
   }
 
+  test("payment request features") {
```

Comment:
test name isn't great

Reply:
Should we test handling of unsupported feature bits

Reply:
Yes to both, good points ;)